### PR TITLE
Gmail manager example agent

### PR DESCRIPTION
**Pull request by TheIsrael1**

### CODE REVIEW

File: examples/agent-chatbot/mastra-logs/agent/asst_2eBHBrQcrJhIsJhN42wa4K91.json
Comment (adeniyii) [inline examples/agent-chatbot/mastra-logs/agent/asst_2eBHBrQcrJhIsJhN42wa4K91.json:1]: this log shouldn't be here

File: examples/agent-chatbot/package.json [hunk 1]
```diff
@@ -12,8 +12,8 @@
   },
   "dependencies": {
     "@ai-sdk/openai": "^0.0.53",
-    "@mastra/core": "^0.1.12",
-    "@mastra/firecrawl": "^1.0.1",
+    "@mastra/core": "workspace:^",
```
Comment (adeniyii) [inline examples/agent-chatbot/package.json]: shouldn't this be pointing to the published version?

File: packages/admin/src/domains/playground/components/api/api-dynamic-form.tsx [hunk 2]
```diff
@@ -165,6 +164,11 @@ function InnerDynamicForm<T extends ZodSchema>({ block }: { block: RefinedIntegr
     }
   }
 
+  console.log({
+    formErrors: errors,
+    schema,
+  });
+
```
Comment (adeniyii) [inline packages/admin/src/domains/playground/components/api/api-dynamic-form.tsx]: stray log?

File: examples/agent-chatbot/mastra-logs/agent/asst_2eBHBrQcrJhIsJhN42wa4K91.json
Comment (TheIsrael1) [reply examples/agent-chatbot/mastra-logs/agent/asst_2eBHBrQcrJhIsJhN42wa4K91.json:1]: hmm

File: examples/agent-chatbot/package.json (hunk 1 above)
Comment (TheIsrael1) [reply examples/agent-chatbot/package.json]: yeah it should, we do need to publish though, i'll revert for now

File: examples/agent-chatbot/mastra-logs/agent/asst_2eBHBrQcrJhIsJhN42wa4K91.json
Comment (TheIsrael1) [reply examples/agent-chatbot/mastra-logs/agent/asst_2eBHBrQcrJhIsJhN42wa4K91.json:1]: i'll investigate why that is

Comment (adeniyii) [reply examples/agent-chatbot/mastra-logs/agent/asst_2eBHBrQcrJhIsJhN42wa4K91.json:1]: yeah i debugged w/ @Joshuafolorunsho  yesterday
there was a manual logger writing to the agent log folder on agent creation

It's been removed now
this log entry was probably created before you pulled latest changes.

Comment (TheIsrael1) [reply examples/agent-chatbot/mastra-logs/agent/asst_2eBHBrQcrJhIsJhN42wa4K91.json:1]: oh alright, i've now removed the log

**Review by adeniyii — APPROVED** [review]

**Pull request merged**